src/app/(proper_react)/(redesign)/(public)/SignUpForm.tsx

```diff
@@ -32,6 +34,15 @@ export const SignUpForm = (props: Props) => {
   const l10n = useL10n();
   const [emailInput, setEmailInput] = useState("");
   const record = useTelemetry();
+  const { view } = props.eventId;
+  const refViewTelemetry = useViewTelemetry(
+    {
+      element_id: view,
+    },
+    {
+      skip: typeof view === "undefined",
```

Comment:
The prop `view` can be `undefined` — skip the creation of `IntersectionObserver` in the `useViewTelemetry` hook.